aws/table_aws_vpc_eip_address_transfer.go

```diff
@@ -0,0 +1,132 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+
+	ec2v1 "github.com/aws/aws-sdk-go/service/ec2"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsVpcEipAddressTransfer(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_vpc_eip_address_transfer",
+		Description: "AWS VPC Elastic IP Address Transfer",
+		List: &plugin.ListConfig{
+			ParentHydrate: listVpcEips,
+			Hydrate:       listVpcEipAddressTransfers,
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "allocation_id", Require: plugin.Optional},
+			},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(ec2v1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "allocation_id",
+				Description: "The allocation ID of an Elastic IP address.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "address_transfer_status",
+				Description: "The Elastic IP address transfer status.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "public_ip",
+				Description: "The Elastic IP address being transferred.",
```

Comment:
Is the description correct?

Reply:
The same description has mentioned in the API response. https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_AddressTransfer.html